collectors/charts.d.plugin/nvme/nvme.chart.sh

```diff
@@ -0,0 +1,141 @@
+# shellcheck shell=bash
+# netdata
+# real-time performance and health monitoring, done right!
+# (C) 2016 Costa Tsaousis <costa@tsaousis.gr>
+#
+# Made by Alexandre Azedo <aazedo@gocontact.pt>
+# 2021
+
+# _update_every is a special variable - it holds the number of seconds
+# between the calls of the _update() function
+nvme_update_every=5
+
+# the priority is used to sort the charts on the dashboard
+# 1 = the first chart
+nvme_priority=49000
+
+# global variables to store our collected data
+critical_warning=
+percentage_used=
+temperature=
+power_cycles=
+power_on_hours=
+
+nvme_get() {
+  # do all the work to collect / calculate the values
+  # for each dimension
+  #
+  # Remember:
+  # 1. KEEP IT SIMPLE AND SHORT
+  # 2. AVOID FORKS (avoid piping commands)
+  # 3. AVOID CALLING TOO MANY EXTERNAL PROGRAMS
+  # 4. USE LOCAL VARIABLES (global variables may overlap with other modules)
+  nvme_list=$(sudo -n nvme list | awk '/nvme/ { print $1 }')
+  local i=
+  for i in $nvme_list;do
+    
+    h=${i/\/dev\//}
+	
+    # read the critical warning in the nvme
+    critical_warning+="SET $h = $(sudo -n nvme smart-log $i | awk -F: '/critical_warning/ {gsub(/ /,""); print $2 }')"$'\n'
+	
```

Comment:
Hello @aazedo ,

I can see that we continue receiving notifications like the above from `shellcheck`. I will give a tip for you, and I hope that after this we will be able to fix this problem and move forward with tests and merging your PR. Please, take a look to this [function](https://github.com/netdata/netdata/blob/9f40c4b12c1047c3489ba07dcf1c1e9b647d1835/health/notifications/alarm-notify.sh.in#L825-L842) that we have inside our alarm notification.

Best regards!

Reply:
Hello @thiagoftsm,
I have already corrected everything according to changes requested by Ferroin. 
Thanks @Ferroin